woocommerce/woocommerce
# #50557 - Order summary Shipping updates

### Changes proposed in this Pull Request:

Closes #50557

* When the address is incomplete we show "Enter address to calculate"
* When the address is provided (but no shipping option available) we show "No available delivery option"
* Font size is changed to 80% of normal and no more bold style, also removed bold style from subtotal as per the design provided in #50557

Before:

 <img width="449" alt="before-no-shipping" src="https://github.com/user-attachments/assets/93b7c172-bb3f-4d88-a301-82bf43e35a9e"> 
 
 After:

Address provided but no shipping option available:
<img width="464" alt="after-no-delivery-option" src="https://github.com/user-attachments/assets/00dd7f53-f6d1-487f-a5f6-b70c7c38e7e4">


No address provided:
<img width="447" alt="after-no-address" src="https://github.com/user-attachments/assets/9f8bc36e-cc99-4607-9205-bb2e40e09370">




### How to test the changes in this Pull Request:

The way I tested this was to:

1. Have 1 shipping option to Netherlands
2. Add products to cart and go to checkout, without an address supplied "Enter address to calculate" should show.
3. Now provide an address but for a country we don't ship to
4. See the copy and styling for "No available delivery option".

Also test these conditions in dark mode, by choosing a dark color palette in the theme editor and change the checkout block to dark mode.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Changes to copy and styling of delivery summary in checkout block.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
